backend/utils.js

```diff
@@ -38,6 +38,8 @@ const capitalize = string => {
   return lowerCase.charAt(0).toUpperCase() + lowerCase.slice(1)
 }
 
+const capitalizeStart = string => string && string[0].toUpperCase() + string.slice(1, string.length)
```

Comment:
Same output as `capitalize`, above?

Reply:
Is it used?

Reply:
Almost, the above lowers the whole string before uppercasing the first character.  Whereas the new one doesn't.